optuna_dashboard/static/components/GraphSlice.tsx

```diff
@@ -0,0 +1,154 @@
+import * as plotly from "plotly.js-dist"
```

Comment:
I replaced `plotly.js-dist` with `plotly.js-basic-dist` at https://github.com/optuna/optuna-dashboard/pull/40 to reduce the file size of bundle.js. Could you merge `master` branch and replace here like below?

```suggestion
import * as plotly from "plotly.js-basic-dist"
```